rdkit-stubs/Chem/ChemicalFeatures.pyi

```diff
@@ -1,4 +1,5 @@
 from rdkit.Chem.rdChemicalFeatures import *
 from rdkit.Chem.rdMolChemicalFeatures import *
+from rdkit.Chem import Mol
 
-def MCFF_GetFeaturesForMol(self, mol, includeOnly: str = ..., confId: int = ...): ...
+def MCFF_GetFeaturesForMol(self, mol: Mol, includeOnly: str = "", confId: int = -1) -> tuple: ...
```

Comment:
Any idea what this tuple contains? Is it `tuple[something, something]`?

Reply:
It seems `tuple[MolChemicalFeature]` and thus I updated.

https://github.com/rdkit/rdkit/blob/4c37b03071181007316c306d874f75aa9f5cb7cf/rdkit/Chem/ChemicalFeatures.py#L14-L19
```py
def MCFF_GetFeaturesForMol(self, mol, includeOnly="", confId=-1):
  res = []
  count = self.GetNumMolFeatures(mol, includeOnly=includeOnly)
  for i in range(count):
    res.append(self.GetMolFeature(mol, i, includeOnly=includeOnly, confId=confId))
  return tuple(res)
```

https://github.com/rdkit/rdkit/blob/4c37b03071181007316c306d874f75aa9f5cb7cf/Code/GraphMol/MolChemicalFeatures/Wrap/MolChemicalFeatureFactory.cpp#L46-L62
```cpp
FeatSPtr getMolFeature(const MolChemicalFeatureFactory &factory,
                       const ROMol &mol, int idx, std::string includeOnly,
                       bool recompute, int confId) {
  static FeatSPtrList feats;
  if (recompute) {
    feats = factory.getFeaturesForMol(mol, includeOnly.c_str(), confId);
  }
  if (idx < 0 || idx >= static_cast<int>(feats.size())) {
    throw IndexErrorException(idx);
  }

  auto fi = feats.begin();
  for (int i = 0; i < idx; ++i) {
    ++fi;
  }
  return (*fi);
}
```

https://github.com/rdkit/rdkit/blob/4c37b03071181007316c306d874f75aa9f5cb7cf/Code/GraphMol/MolChemicalFeatures/MolChemicalFeatureFactory.h#L20-L21
```cpp
typedef boost::shared_ptr<MolChemicalFeature> FeatSPtr;
typedef std::list<FeatSPtr> FeatSPtrList;
```

Reply:
Looks like a tuple with a variable number of elements. So it would be `tuple[MolChemicalFeature, ...]` I think.

Reply:
Indeed. Thank you for pointing out!